CONTRIBUTING.md

```diff
@@ -107,6 +110,12 @@ adjust the proof indentation to match any indentation changes made by /rewrap.
 Then, 'verify markup' will check that no lines became too long due to different
 indentation.  Finally, 'verify proof' is there because you might as well.
 
+In addition, you are encouraged to run `scripts/minimize label`, where label is
+the name of your proof. This script attempts to to shorten the proof using other
```

Comment:
Typo: to to

Reply:
Thanks, fixed.

Reply:
Maybe " name of your theorem" instead of  "name of your proof".